qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/IndexingIT.java

```diff
@@ -5,27 +5,123 @@
 
 package org.opensearch.knn.bwc;
 
+import org.opensearch.common.Strings;
+import org.opensearch.common.xcontent.XContentFactory;
+import org.opensearch.knn.index.SpaceType;
+
+import java.util.Optional;
+
+import static org.opensearch.knn.TestUtils.KNN_ALGO_PARAM_EF_CONSTRUCTION_MIN_VALUE;
+import static org.opensearch.knn.TestUtils.KNN_ALGO_PARAM_M_MIN_VALUE;
+import static org.opensearch.knn.TestUtils.KNN_ENGINE_FAISS;
 import static org.opensearch.knn.TestUtils.NODES_BWC_CLUSTER;
+import static org.opensearch.knn.common.KNNConstants.KNN_METHOD;
+import static org.opensearch.knn.common.KNNConstants.METHOD_HNSW;
+import static org.opensearch.knn.common.KNNConstants.NAME;
+import static org.opensearch.knn.common.KNNConstants.PARAMETERS;
 
 public class IndexingIT extends AbstractRestartUpgradeTestCase {
     private static final String TEST_FIELD = "test-field";
     private static final int DIMENSIONS = 5;
     private static final int K = 5;
-    private static final int ADD_DOCS_CNT = 10;
+    private static final int ADD_DOCS_COUNT = 10;
```

Comment:
nit: NUM_DOCS

Reply:
I'm using ADD_DOCS_COUNT for the number of documents to add into the index. If I rename it as NUM_DOCS, it might create some confusion between adding docs count and search docs count.

Reply:
Usually searches are referred to as queries, so like QUERY_COUNT or NUM_QUERIES. DOCS are related to INDEX.

Reply:
makes sense